config/index.js

```diff
@@ -285,5 +288,46 @@ module.exports = {
     'wrap-regex': 'error',
     'yield-star-spacing': ['error', 'both'],
     'yoda': ['error', 'never'],
+
+    'import/default': 'error',
+    'import/dynamic-import-chunkname': 'off',
+    'import/export': 'error',
+    'import/exports-last': 'off',
+    'import/extensions': ['error', 'never', { 'json': 'always' }],
+    'import/first': 'error',
+    'import/group-exports': 'off',
+    'import/max-dependencies': 'off',
+    'import/named': 'error',
+    'import/namespace': 'error',
+    'import/newline-after-import': ['error', { 'count': 1 }],
+    'import/no-absolute-path': 'error',
+    'import/no-amd': 'off',
+    'import/no-anonymous-default-export': ['error', { 'allowObject': true }],
+    'import/no-commonjs': 'off',
+    'import/no-cycle': 'off',
+    'import/no-default-export': 'off',
+    'import/no-deprecated': 'off',
+    'import/no-duplicates': 'error',
+    'import/no-dynamic-require': 'error',
+    'import/no-extraneous-dependencies': 'off',
+    'import/no-internal-modules': 'off',
+    'import/no-mutable-exports': 'error',
+    'import/no-named-as-default': 'error',
+    'import/no-named-as-default-member': 'error',
+    'import/no-named-default': 'error',
+    'import/no-named-export': 'off',
+    'import/no-namespace': 'off',
+    'import/no-nodejs-modules': 'off',
+    'import/no-relative-parent-imports': 'off',
+    'import/no-restricted-paths': 'off',
+    'import/no-self-import': 'error',
+    'import/no-unassigned-import': 'error',
+    'import/no-unresolved': ['error', { 'commonjs': true }],
+    'import/no-unused-modules': 'off',
+    'import/no-useless-path-segments': ['error', { 'commonjs': true, 'noUselessIndex': true }],
+    'import/no-webpack-loader-syntax': 'error',
+    'import/order': 'off',
+    'import/prefer-default-export': 'off',
+    'import/unambiguous': 'off',
```

Comment:
This seems useful - mostly because the import plugin itself might not work correctly if these ambiguities exist.

Reply:
Enabled!